Sources/SmileID/Classes/DocumentVerification/View/DocumentCaptureView.swift

```diff
@@ -42,9 +42,11 @@ struct DocumentCaptureView: View {
                             .foregroundColor(SmileID.theme.accent)
                             .frame(width: 235, alignment: .center)
                     }
-                    CaptureButton {
-                        viewModel.captureImage()
-                    }.padding(.bottom, 60)
+                    if viewModel.borderColor == SmileID.theme.success.uiColor() {
+                        CaptureButton {
+                            viewModel.captureImage()
+                        }.padding(.bottom, 60)
+                    }
```

Comment:
The capture button visibility should be its own property -- the eventual scenario will be that the capture button is not visible for the first 10 seconds, even though the border is green. And also, we wouldn't want to show the button when a capture is in progress but we still want to show a green border